packages/chrome/src/elements/nav/NavItem.tsx

```diff
@@ -13,21 +13,13 @@ import { useNavContext } from '../../utils/useNavContext';
 import { useChromeContext } from '../../utils/useChromeContext';
 
 interface INavItemProps extends HTMLAttributes<any> {
-  /**
-   * Applies product-specific color palette
-   **/
+  /** Sets a product-specific color palette */
   product?: PRODUCT;
-  /**
-   * Indicate which item is current in the nav
-   **/
+  /** Determines if the current item is in the navigation */
   isCurrent?: boolean;
-  /**
-   * Indicate that the item contains a product logo
-   */
+  /** Determines if the item contains a product logo and prepares the item content to display the logo */
```

Comment:
Determines if the item contains a product logo ~and prepares the item content to display the logo~

Reply:
Fixed in latest update.